### ISSUE TITLE
A11y_Code.visualstudio.com_visual studio Learn page_Overview_Usability: Inside dropdown the items are visually looks like a text but programmatically defined as a link.

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5159]
## GitHub Tags:
#A11yTCS;#Win10;#DesktopWeb;#A11ySev4;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;#A11yUsable

## Environmental Details:
#URL: https://code.visualstudio.com/learn

## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version 10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)

## Repro Steps:

1. Hit the URL: https://code.visualstudio.com/learn
2. Tab till any dropdown and press enter.
3. Observe the dropdown  whether the items are visually looks like a text but programmatically defined as a link  or not.

## Note: 
Same issue found in this URL: https://code.visualstudio.com/blogs

## Actual results:
Inside dropdown the items are visually looks like a text but programmatically defined as a link.

## Expected results:
Inside dropdown the items should visually look like a link and also programmatically it should be defined as a link.

## User Impact:
Users will get confuse to differentiate whether they are text or links.

## Attachments:

https://user-images.githubusercontent.com/99243079/153196026-b0eb21dd-9e38-413a-9efc-6d0b5e0e78f9.mp4

![Bug_5159_visual studio Learn page_Overview_Usability](https://user-images.githubusercontent.com/99243079/153196070-8683edba-7c12-4d69-ac38-f379071ffb14.png)

https://user-images.githubusercontent.com/99243079/153401847-5027ff90-5722-4602-861f-722974aeed89.mp4

**Comment by Samhitha66** [comment]
@rzhao271 @gregvanl 

Checked again this bug in the below environment
For main issue URL: https://code.visualstudio.com/learn
For note issue URL: https://code.visualstudio.com/blogs

The main issue is fixed, the note issue is still reproving.
So, please reopen this bug.

https://user-images.githubusercontent.com/99243079/157206063-fd73d881-fadb-4b1f-88cc-40de83f7e0fe.mp4


https://user-images.githubusercontent.com/99243079/157206084-7715105e-a7e7-4074-a45c-183313ce6c5c.mp4

**Comment by Samhitha66** [comment]
@gregvanl 

Checked again this bug in the below environment
For note issue URL: https://code.visualstudio.com/blogs
The  note issue is also fixed.

https://user-images.githubusercontent.com/99243079/158576332-59aef970-27f5-4693-8840-d7543750dc7e.mp4
